decisions/importer/helsinki.py

```diff
@@ -43,6 +47,10 @@
     13: 'Kaupunki',
     14: 'Yksikkö',
     15: 'Toimikunta',
+    16: 'Palvelukokonaisuus',
```

Comment:
This should be `Koulujen johtokunnat` and number 17 should be `Palvelukokonaisuus`